procrastinate/aiopg_connector.py

```diff
@@ -221,10 +218,18 @@ def _make_dynamic_query(self, query: str, **identifiers: str) -> str:
     @wrap_exceptions
     async def listen_notify(
         self, event: asyncio.Event, channels: Iterable[str]
-    ) -> NoReturn:
+    ) -> None:
         pool = await self._get_pool()
         # We need to acquire a dedicated connection, and use the listen
         # query
+        if pool.maxsize == 1:
+            logger.warning(
+                "Listen/Notify capabilities disabled because maximum pull size"
+                "is set to 1",
+                extra={"action": "listen_pool_too_small"},
```

Comment:
Why `listen_pool_too_small` here? Do you mean `pool_too_small` instead?

Reply:
ok, but I want toconvey the idea that it's about listen/notify. An idea ?

Reply:
pool_too_small_for_listen_notify ?